src/model.rs

```diff
@@ -989,6 +989,41 @@ impl Collections {
         self.stop_areas = CollectionWithId::new(updated_stop_areas).unwrap();
     }
 
+    /// Check that all references to geometries actually points towards existing
+    /// `Geometry`. This is a common problem where a `geometries.txt` is read in
+    /// NTFS, a line of this file is not a valid WKT format, then the `Geometry`
+    /// is not created. However, the object that references this `Geometry` will
+    /// very likely be created (for example, a `Route` from `routes.txt`). This
+    /// creates a incoherent model where a `Route` points to a `Geometry` which
+    /// doesn't exist.
+    ///
+    /// This function checks that all objects points to existing `Geometry` and,
+    /// in the case it doesn't, fix the model by removing this pointer.
+    fn check_geometries_coherence(&mut self) {
+        macro_rules! check_and_fix_object_geometries {
+            ($collection:expr) => {
+                let objects_to_fix: Vec<String> = $collection
+                    .values()
+                    .filter(|line| {
```

Comment:
I suggest to use obj instead of line as it also concerns others objects

Reply:
Good point. Done in eaef491376a016ac4aec5be7d4f99b3116174621.